core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/factory/Factory.java

```diff
@@ -32,10 +32,7 @@
   /** Returns creator of this factory instance. */
   Author getCreator();
 
-  /**
-   * Returns a workspace configuration of this factory instance, it is mandatory for every factory
-   * instance.
-   */
+  /** Returns a workspace configuration of this factory instance. */
```

Comment:
I believe it's still a mandatory field for every factory instance. 
It may be nullable only for DTOs